week-life-front-end/src/components/Authentication.js

```diff
@@ -5,10 +5,14 @@ import { Logout } from './Logout';
 
 
 export const Authentication = () => {
+
+
+
+    console.log("componente autentication creado")
     const clientId = "869435522440-9c7q2p1ao5ki97orc02pd7nnjduq1gtn.apps.googleusercontent.com"
```

Comment:
avoid hard-coding values like this, use env variables
https://create-react-app.dev/docs/adding-custom-environment-variables/